src/commands/moderation/anti-raid-nuke.ts

```diff
@@ -39,250 +36,31 @@ export default class implements Command {
 			throw new Error(i18next.t('common.errors.no_mod_log_channel', { lng: locale }));
 		}
 
-		const parsedJoin = ms(args.join);
-		if (parsedJoin < 6000 || isNaN(parsedJoin)) {
-			throw new Error(i18next.t('command.common.errors.duration_format', { lng: locale }));
-		}
-
-		const parsedAge = ms(args.age);
-		if (parsedAge < 6000 || isNaN(parsedAge)) {
-			throw new Error(i18next.t('command.common.errors.duration_format', { lng: locale }));
-		}
-
-		if (args.reason && args.reason.length >= 500) {
-			throw new Error(i18next.t('command.mod.common.errors.max_length_reason', { lng: locale }));
+		const archiveChannel = await checkLogChannel(
+			interaction.guild,
+			(await getGuildSetting(interaction.guildId, SettingsKeys.GeneralLogChannelId)) as string,
+		);
+		if (!archiveChannel) {
+			throw new Error(i18next.t('common.errors.no_general_log_channel', { lng: locale }));
 		}
 
-		const joinCutoff = Date.now() - parsedJoin;
-		const accountCutoff = Date.now() - parsedAge;
+		const modRoleId = (await getGuildSetting(interaction.guildId, SettingsKeys.ModRoleId)) as string;
```

Comment:
prefer ban member permission checks over modrole id (deprecated since we switched to slash commands)